totalRP3/Modules/TargetFrame/TargetFrame.lua

```diff
@@ -177,24 +177,22 @@ local function onStart()
 		return profile;
 	end
 
-	local TARGET_NAME_WIDTH = 168;
```

Comment:
I'm assuming this was removed because the default value of 150 is close enough?

Reply:
Something along those lines yeah.